src/index.js

```diff
@@ -1,13 +1,13 @@
 var isArray = Array.isArray
 
 export default function cc(obj) {
-  var out = ""
-
-  if (typeof obj === "string" || typeof obj === "number") return obj || ""
+  if (typeof obj === "string") return obj
+  else if (typeof obj === "number") return obj.toString()
```

Comment:
Let's use `(obj + "")` instead. 🙏

Reply:
Or even better with less code:

```js
if (typeof str === "string" || typeof str === "number") return str + ""
```

Reply:
My approach avoids unnecessary string concatenation, but I agree with you that bundle size is relevant, so I'll change it to `"" + str` (starting with `""` because it matches the return type).